user/cron-jobs.md

```diff
@@ -27,6 +25,10 @@ Confirm that the cron job is displayed in your settings tab:
 
 ![cron job created](/images/cron-created.png "cron job created")
 
+## Skipping Cron Jobs
+
+Because cron jobs build the latest commit to their branch, if a commit message includes [`[ci skip]` or `[skip ci]`](/user/customizing-the-build/#Skipping-a-build) it will not be built. 
```

Comment:
YES! I have *no idea* why it took me so long to write that sentence, but it still felt weird when I pushed it out. This fixes it.